standards/core/store-sync.md

```diff
@@ -0,0 +1,52 @@
+---
+title: WAKU-STORE-SYNC
+name: Waku Store Synchronization
+editor: Simon-Pierre Vivier <simvivier@status.im>
+contributors:
+---
+
+## Abstract
+
+This document describe the strategy Waku Store node will employ to stay synchronized.
+The goal being that all store nodes eventually archive the same set of messages.
+
+## Background / Rationale / Motivation
+
+Message propagation in the network is not perfect,
+even with GossipSub mechanisms to detect missed messages.
+Nodes can also go offline for various reason outside our control.
+Store nodes that want to provide a good service must be able to remedy situations like these.
+By having store nodes syncronize with each other through various protocols,
+the set of archived messages network wide will be eventually consistent.
+
+## Theory / Semantics
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “NOT RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Various protocols and features that help with message consistency are described below.
+
+### Store Resume
+
+This feature allow a node to fill the gap in messages for the period it was last offline.
+At startup, a node use the Store protocol to query a random node for
+the time interval since it was last online.
```

Comment:
Perhaps only for future versions of the spec, but restating this as a heuristic rather than a high-level description of functionality allows the spec to be useful for new implementers, e.g.:

On startup, all Store Sync nodes SHOULD perform the resume procedure as set out below.
On startup the node MUST determine the last time it was online.
The node SHOULD do this by reading.... (?)
The Store Sync node MUST then craft a new store query to a random Store node
to retrieve all messages on relevant shards since the last time it was online.
The node MUST subsequently add these to the local storage.